migrate.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+# Usage: ./migrate.sh -s <source_environment> -t <target_environment> [-r]
```

Comment:
I think having a confirmation step might be useful since this could be quite a destructive action.  E.g:
"You are migrating the current development environment into the production environment. The production environment schema will be updated as a result. Is this correct? (y/n)"

Reply:
I'll add more confirmation prompts